python_modules/libraries/dagster-databricks/dagster_databricks_tests/test_pyspark.py

```diff
@@ -0,0 +1,240 @@
+import os
+
+import pytest
+
+from dagster_aws.s3 import s3_plus_default_storage_defs, s3_resource
+from dagster_databricks import databricks_pyspark_step_launcher
+from dagster_pyspark import DataFrame, pyspark_resource
+from pyspark.sql import Row
+from pyspark.sql.types import IntegerType, StringType, StructField, StructType
+
+from dagster import (
+    InputDefinition,
+    ModeDefinition,
+    OutputDefinition,
+    execute_pipeline,
+    pipeline,
+    reconstructable,
+    solid,
+)
+from dagster.core.definitions.no_step_launcher import no_step_launcher
+from dagster.seven import mock
+from dagster.utils.merger import deep_merge_dicts
+
+# TODO: uncomment when dagster-azure is merged.
```

Comment:
Is there currently outstanding work on dagster-azure?

Reply:
Yeah, I'll tidy it up today and send a PR, it needs a bunch of tests adding though...

Reply:
Yeah, I'll tidy it up today and send a PR, it needs a bunch of tests adding though...

Reply:
A WIP PR is here: https://github.com/dagster-io/dagster/pull/2483.